src/editor/editor.directive.ts

```diff
@@ -166,28 +167,43 @@ export class FroalaEditorDirective implements ControlValueAccessor {
     if (!this._opts.events) {
       this._opts.events = {};
     }
+
+    // Store user event callbacks if exist
+    if (this._opts.events[eventName]) {
```

Comment:
@akki744 why are we creating copy. What if user decides to change the callbacks we won't be able to get updated reference. Can't we execute directly from props?

Reply:
@navneetnagpal Updated the code to handle this.